test/Core.js

```diff
@@ -19,38 +19,205 @@
 //
 // ----------------------------------------------------------------------------
 
-const Core_utils = require('./Core_utils.js');
-
-///
-/// Test stories
-/// 
-/// Properties
-/// 	has coreRegistrar
-/// 	has coreChainIdRemote
-/// 	has coreOpenSTRemote
-/// 
-
-contract('Core', function(accounts) {
-	const registrar = accounts[1];
-	const chainIdRemote = 1410;
-	const openSTRemote = accounts[4];
-
-	describe('Properties', async () => {
-		before(async () => {
-	        contracts = await Core_utils.deployCore(artifacts, accounts);
-	        core = contracts.core;
-	    })
-
-		it('has coreRegistrar', async () => {
-			assert.equal(await core.registrar.call(), registrar);
-		})
-
-		it('has coreChainIdRemote', async () => {
-			assert.equal(await core.chainIdRemote.call(), chainIdRemote);
-		})
-
-		it('has coreOpenSTRemote', async () => {
-			assert.equal(await core.openSTRemote.call(), openSTRemote);
-		})
-	})
-})
+const coreUtils = require('./Core_utils.js')
+    , utils = require('./lib/utils.js')
+    , proof = require('./data/proof')
+    , ethUtil = require('ethereumjs-util')
+    , BigNumber = require('bignumber.js')
+    , web3EventsDecoder = require('./lib/event_decoder.js')
+;
+
+contract('Core', function (accounts) {
+    const openSTRemote = proof.account.openSTRemoteAddress
+        , blockHeight = new BigNumber(5)
+    ;
+
+    describe('Properties', async () => {
+        before(async () => {
+            contractsData = await coreUtils.deployCore(artifacts, accounts);
+            core = contractsData.core;
+            workerContract = contractsData.workerContract;
+            worker = contractsData.worker;
+            registrar = contractsData.registrar;
+            chainIdRemote = contractsData.chainIdRemote;
+            chainIdOrigin = contractsData.chainIdOrigin;
+            await core.commitStateRoot(blockHeight.toNumber(), proof.account.stateRoot, {from: worker});
+            await core.proveOpenST(blockHeight.toNumber(), proof.account.rlpEncodedAccount, proof.account.rlpParentNodes, {from: worker});
```

Comment:
core.commitStateRoot is needed for core.getLatestStateRootBlockHeight property test since it's populated in this method.
core.proveOpenST is not needed. 
Updated the code accordingly.